share/sysbuild/CMakeLists.txt

```diff
@@ -68,6 +73,33 @@ add_subdirectory(bootloader)
 
 # This allows for board and app specific images to be included.
 include(${BOARD_DIR}/sysbuild.cmake OPTIONAL)
-include(${APP_DIR}/sysbuild.cmake OPTIONAL)
+
+# Images must be processed in a nested loop to allow adding additional targets
+get_property(ITERATE_IMAGES GLOBAL PROPERTY ITERATE_IMAGES)
+get_property(ITERATE_IMAGE_DIRS GLOBAL PROPERTY ITERATE_IMAGE_DIRS)
+list(LENGTH ITERATE_IMAGES ITERATE_IMAGE_SIZE)
+while(ITERATE_IMAGE_SIZE GREATER 0)
```

Comment:
I believe this can be made simpler by avoiding two lists `IMAGES` and `ITERATE_IMAGES`.
As well as not relying on global property calls.

Let me try to make a proposal.

Reply:
Please find a much simpler approach which uses the existing `IMAGES` list and just create a processed list.

No need for any additional lists and no need for changes to `ExternalZephyrProject_Add()`.
https://github.com/nordicjm/zephyr/pull/1

Reply:
Fixed with PR